packages/gatsby-theme-docs/src/layouts/internals/page-content-inset.js

```diff
@@ -16,7 +16,7 @@ const PageContentInset = styled.div`
   max-width: ${(props) =>
     props.maxWidth || designSystem.dimensions.widths.pageContentWide};
   section {
-    overflow: auto;
+    overflow: clip;
```

Comment:
`overflow: auto` has been added by this PR: https://github.com/commercetools/commercetools-docs-kit/pull/1420/files

I've verified that the active selection still works as expected.

Reason behind `clip` value:
CSS `position: sticky` works only when there's no overflow restrictions applied at any level in the ancestors. see https://www.terluinwebdesign.nl/en/css/position-sticky-not-working-try-overflow-clip-not-overflow-hidden/

As possible solution `clip` is a sort of middle ground that seems to be working solving both the issues (active section and sticky content)

Reply:
nice research.  Please consider documenting the rationale as a code comment. This is the kind of things that get really hard to re-engineer if later someone else touches the CSS.

Reply:
Update:  in most pages it's an increased spacing when an H2 heading is directly following an H1 heading.  But there's also one case in the images docs where it's in a place where an H2 heading is after an image.

Reply:
@nkuehn The VRT changes you see are basically reverting the VRT changes introduced with the PR regarding selected section. (the cause is the change of the overflow property).
As you can see here are the original VRT changes:
https://percy.io/commercetools-GmbH/docs-kit-smoke-tests/builds/23092707/changed/1289339663?browser=chrome&browser_ids=31&subcategories=approved&viewLayout=side-by-side&viewMode=new&width=1706&widths=1706

and here the latest:
https://percy.io/commercetools-GmbH/docs-kit-smoke-tests/builds/23496946/changed/1313464350?browser=chrome&browser_ids=31&subcategories=unreviewed%2Cchanges_requested&viewLayout=side-by-side&viewMode=new&width=1706&widths=1706

Basically we're reverting the UI changes but the functionality still works as expected